fbpic/lpa_utils/laser/antenna_injection.py

```diff
@@ -239,6 +247,11 @@ def update_v( self, t ):
         t: float (seconds)
             The time at which to calculate the velocities
         """
+        # Interrupt this function if the antenna is not currently
+        # active on the global domain (as determined by `update_current_rank`)
+        if not self.active_update_v:
+            return
+
```

Comment:
```suggestion
        # Interrupt this function if the antenna is not currently
        # active on the global domain
        zmin_global, zmax_global = comm.get_zmin_zmax(
            local=False, with_damp=True, with_guard=True )
        if (z_antenna >= zmin_global) and (z_antenna < zmax_global):
            return

```

Reply:
well this would `return` for antenna in domain =)

```suggestion
        # Interrupt this function if the antenna is not currently
        # active on the global domain
        zmin_global, zmax_global = comm.get_zmin_zmax(
            local=False, with_damp=True, with_guard=True )
        if (z_antenna < zmin_global) or (z_antenna > zmax_global):
            return
```